sunpy_sphinx_theme/conf.py

```diff
@@ -66,23 +57,12 @@ def page_url(page):
             ],
             1,
         ),
-        ("Blog", page_url("blog.html"), 1),
-        ("Support Us", page_url("contribute.html"), 1),
         ("Get Help", page_url("help.html"), 1),
-        (
-            "SunPy Project",
-            [
-                ("SunPy Project", page_url("project/"), 1),
-                ("Community Roles", page_url("project/roles.html"), 1),
-                ("Affiliated Packages", page_url("project/affiliated.html"), 1),
-                ("Emeritus role holders", page_url("project/former.html"), 1),
-                ("Meetings", page_url("project/meetings.html"), 1),
-            ],
-            1,
-        ),
```

Comment:
This has to stay like this, it's a nested folder in the repo and I can't recall if it's possible to link it under a dropdown as things stand.

Reply:
Tho maybe we can get it to link to the index.html page, we can get this to work.

Reply:
Where has this gone?